docs/test.md

```diff
@@ -79,19 +79,47 @@ Lastly, build the actual QA image.
 $ docker build -t tritonserver_qa -f Dockerfile.QA .
 ```
 
-## Run QA Container
+## Run QA Tests
 
-Now run the QA container and mount the QA model repositories into the
+Now run the QA image and mount the QA model repositories into the
 container so the tests will be able to access them.
 
 ```
 $ docker run --gpus=all -it --rm -v/tmp:/data/inferenceserver tritonserver_qa
 ```
 
 Within the container the QA tests are in /opt/tritonserver/qa. To run
-a test simply change directory to the test and run the test.sh script.
+a test, change directory to the test and run the test.sh script.
 
 ```
 $ cd <test directory>
 $ bash -x ./test.sh
 ```
+
+### Sanity Tests
+
+Many tests require that you use a complete Triton build, with all
+backends and other features enabled. There are three sanity tests that
+are parameterized so that you can run them even if you have built a
+Triton that contains only a subset of Triton backends. These tests are
+L0_infer, L0_batcher and L0_sequence_batcher. For these tests the
+following envvars are available to control how the tests behave:
+
+* BACKENDS: Control which backends are tested. Look in the test.sh
+  file of the test to see the default and allowed values.
+
+* ENSEMBLES: Enable testing of ensembles. Set to "0" to disable, set
+  to "1" to enable. If enabled you must have the *identity* backend
+  included in your Triton build.
+
+* EXPECTED_NUM_TESTS: The tests perform a check of the total number of
+  test sub-cases. The exact number of sub-cases that run will depend
+  on the values you use for BACKENDS and ENSEMBLES. So you will need
+  to adjust this as appropriate for your testing.
+
+For example, if you build a Triton that has only the TensorRT backend
+you can run L0_infer as follows:
+
+```
+$ BACKENDS="plan" ENSEMBLES=0 bash -x ./test.sh
```

Comment:
Should give `EXPECTED_NUM_TESTS` in the example as well since it will not be the usual number if only TRT is enabled.

Reply:
I don't know what the correct value is and it may change over time. But I will add some additional text.